components/input/index.en-US.md

```diff
@@ -26,7 +26,7 @@ Keyboard and mouse can be used for providing or changing data.
 | prefix | The prefix icon for the Input. | string\|ReactNode |  |
 | size | The size of the input box. Note: in the context of a form, the `large` size is used. Available: `large` `default` `small` | string | `default` |
 | suffix | The suffix icon for the Input. | string\|ReactNode |  |
-| type | The type of input, `text` or `textarea`(`type=textarea` are deprecated after `2.12`, please use `Input.TextArea`) | string | `text` |
+| type | The type of input (use `Input.TextArea` instead of `type="textarea"`) | string | `text` |
```

Comment:
So can the `type` attribute be entirely removed in 3.0?

Reply:
`type` has other values like `password`, `email` etc.

Reply:
Ahhh, right. Would it be worth including the types that are valid (I think the `use `Input.TextArea` is worth including!)

Reply:
It's a long list, and `type` is just a standard HTML `attribute`, so I'm keeping it as it is.

Reply:
👍 maybe a link to https://developer.mozilla.org/docs/Web/HTML/Element/input#Form_%3Cinput%3E_types would be worth referencing

Reply:
👍 Added.